src/libs/styles.c

```diff
@@ -536,7 +642,7 @@ void gui_init(dt_lib_module_t *self)
   /* filter entry */
   w = gtk_entry_new();
   d->entry = GTK_ENTRY(w);
-  gtk_entry_set_placeholder_text(GTK_ENTRY(d->entry), _("filter styles name"));
+  gtk_entry_set_placeholder_text(GTK_ENTRY(d->entry), _("filter style names"));
```

Comment:
Also you want to rebase  your branch on top of current master as this change is already committed.